### ISSUE TITLE
"Move to new file" does not copy namespace imports

### ISSUE THREAD

**Issue by OliverJAsh** [microsoft/TypeScript#33156]
**TypeScript Version:**  3.5.2, 3.6.2

**Search Terms:**

**Code**

```ts
// ./src/foo.ts
export const whatever = 1;
```

```ts
// ./src/index.ts
import * as foo from './foo';

const bar = foo.whatever;

export default bar;
```

**Expected behavior:**
When copying the line `const bar` and actioning the "Move to new file" command, the new file contents should be:

``` tsx
import * as foo from './foo';
export const bar = foo.whatever;
```

**Actual behavior:**
The new file contents are:

``` tsx
export const bar = foo.whatever;
```

The `foo` namespace import is missing, which creates errors:

![image](https://user-images.githubusercontent.com/921609/64011416-8fddb680-cb13-11e9-9d9d-5a71d42cde62.png)

**Playground Link:** 

**Related Issues:** https://github.com/microsoft/TypeScript/issues/27937, which was apparently fixed but it seems to be happening again.

**Comment by nmain** [comment]
I encountered this today and got 99% of the way through writing a bug report before I found the duplicate :cry:.  This still happens in typescript@3.9.0-dev.20200309

I'll say "TS2686" now to add that search term, as it's a common error to get on the destination side if the missing import is something like `import * as ReactDOM from "react-dom"`.

**Comment by jakebailey** [comment]
This was fixed by #38377; #35591 was a dupe of this issue.
